renovate.json5

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": [
+    "config:js-lib",
+    // Our default configuration. See
+    // https://github.com/apollographql/renovate-config-apollo-open-source/blob/master/package.json
+    "apollo-open-source",
+  ],
+  "schedule": null,
+  "prCreation": "immediate",
+  "packageRules": [
+    // Bunch up all non-major dependencies into a single PR.  In the common case
+    // where the upgrades apply cleanly, this causes less noise and is resolved faster
+    // than starting a bunch of upgrades in parallel for what may turn out to be
+    // a suite of related packages all released at once.
+    {
+      "groupName": "all non-major dependencies",
+      "matchUpdateTypes": ["patch", "minor"],
+      "groupSlug": "all-minor-patch",
+    },
+    // We set this to the lowest supported Node.js version to ensure we don't
+    // use newer Node.js APIs unknowingly during development which are going to
+    // fail in CI anyway when they're run against the full range of Node.js
+    // versions we support.
+    {
+      "matchPackageNames": ["@types/node"],
+      "allowedVersions": "12.x"
+    },
```

Comment:
💭 I don't think this applies to us since our code runs on the client and is transpiled